guides/common/modules/con_cloning-project-server.adoc

```diff
@@ -0,0 +1,24 @@
+:_mod-docs-content-type: CONCEPT
+
+[id="cloning_{project-context}_server"]
+= Cloning {ProjectServer}
+
+You can clone your {ProjectServer} to create instances to test upgrades and migration of instances to a different machine or operating system.
+This is an optional step to provide more flexibility during the upgrade or migration.
+
+You cannot use the {Project} clone tool on a {SmartProxyServer}.
+Instead, you must backup the existing {SmartProxyServer}, restore it on the target server, and then reconfigure {SmartProxyServer}.
+
+Note that because of the high volume of data that you must copy and transfer as part of these procedures, it can take a significant amount of time to complete.
```

Comment:
```suggestion
Note that because of the high volume of data that you must copy and transfer as part of the cloning process, it can take a significant amount of time to complete.
```
Avoid "procedures".